### ISSUE TITLE
Section 2 Feedback Through Section 2.6

### ISSUE THREAD

**Issue by michelnev** [w3c/webmediaguidelines#47]
Please feel free to shoot holes in any of these suggestions:

1. In the first paragraph of Content Generation: I suggest adding mezzanine file to the glossary.  I'd suggest eliminating the use of "Original Content" and just use mezzanine file since "original content" is also used in section 3.3.1 to indicate the content to be played after leaving an Ad.  The first sentence says, "discussed in section below", but that could probably be removed (I'm not sure what it is referring to).

2. Suggest rewording second paragraph in Content Generation.  Maybe something like:  "ABR streaming content is generated by encoding the mezzanine file against an encoding profile. Firstly, the mezzanine file is encoded into a set of versions, each with its own average bitrate. Secondly, each version is packaged into segments of a specified duration."  Note, I prefer transcoded to encoded in this context, but the meaning is clear either way.

3. Suggest minor changes to the final paragraph in Content Generation. (didn't find a way to do strike-through, so some text has been removed)  "The second process is performed by a packager which segments the different bitrates. These are then packaged into a transport format such as transport streams (.ts) or fragmented MP4s (.m4s)**_._** **_Next_**, they are **_optionally_** encrypted with a DRM that is suitable for the environment where the content is going to be played out.  The packager is also responsible for the generation of a manifest file, typically a DASH (.mpd), HLS (.m3u8), Smooth (.ism) or HDS (.f4v), that specifies the location of the media and its format."

4. In the second paragraph of Content Delivery: There is a little too much use of the word "it".  In the last few sentences, "it" changes from referencing the CDN to referencing the manifest.  I'd suggest something like this:   "The player makes a HTTP GET request for the manifest from CDN edge, the edge server's location is determined via DNS.  If the CDN edge does not currently have the manifest available, the CDN requests a copy of the file from the origin and the file is cached within the server for later use.  The CDN edge then returns the the manifest to the requesting player."  

5. In the Content Playback section: number 1: Is there a difference between a client and a player?  Maybe start the sentence with "The _player_ uses **it's** DRM license URL..."

6. In the Content Playback section: number 2: Since it is in the glossary, can you get rid of "(adaptive bitrate)"?  Could we clarify this section to describe it as a network throughput calculation?  Maybe something like: "The player's ABR algorithm determines the data throughput rate available to the client by calculating the throughput from the first segment of video. This provides enough information to determine the initial playback quality level the player can sustain."

7. In the Content Playback section: number 3: Maybe clarify what "this information" is?  Change "available bitrate" to "calculated throughput"?

8. In the Content Playback section: number 4: Slight change: "The player then requests a segment from the edge server, which is typically relative to the location of the manifest."

9.  In the Content Playback section: number 5: slight change: "If encrypted, the segment is decrypted in accordance with the specific DRM in use."

10. In the Content Playback section: number 6: Make it clear what the "it" words reference. Something like "The player then adds the segment to the video buffer."

11. In the Content Playback section: number 7: Is "media engine" standard terminology?  I've used terms like video pipeline or decoding pipeline.   Maybe something like: "The media engine decodes the data and passes it to the video surface where it is rendered."

12. In the Content Playback section: number 8: In previous bullet points, "video quality" was used instead of "bitrate stream".  I suggest referring to it in one way.  I personally think bitrate is more accurate than quality.  Also, there are some missing commas: "If the **throughput** remains constant**_,_** the player will continue to request segments of the same bitrate stream **as previoulsy defined.** In the event of a change in network **throughput****_,_** the player will make a decision about the need to either drop to a lower bitrate stream or a higher bitrate stream."

13. In the On-Demand Streaming (VOD) section, the sentence is a little awkward.  Maybe something like this: " Despite the almost identical mechanics, content generation, delivery and play out, used for VOD and linear, a large organization will typically maintain two distinct workflows as there are subtle but important ways in which they differ. "  If feels like the following sections 2.6 - 2.8 and maybe 2.9 and 2.10, should be subsection of 2.5.

14. In the 2.6 Content Generation section, there are some missing commas: "For VOD,...", "To this end,...", "As we will see shortly,..."

**Comment by joelkorpi** [comment]
Great feedback. Thank you!

**Comment by joelkorpi** [comment]
#1 - Clarify "original content" vs. "mezzanine file" in "Content Generation" - clarify that mezzanine is optional in workflow
#6 - 2.8 add "ABR algorithm" to glossary; clarify that not all implementations of ABR are based on thruput calculations
#11 - 2.4 change "media engine" to "media renderer" (add "media renderer" to glossary)
#12 - conform to "rendition" from "video quality" etc.

**Comment by joelkorpi** [comment]
Opened PR https://github.com/w3c/webmediaguidelines/pull/65
